dir.targets

```diff
@@ -52,15 +57,21 @@
 
     <!-- Restore build tools -->
     <Exec Command="$(NugetRestoreCommand) &quot;$(SourceDir).nuget\packages.config&quot;" StandardOutputImportance="Low" />
-    
-    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true'"
-           Text="The build tools package was just restored and so we cannot continue the build of an individual project because targets from the build tools package were not able to be imported. Please retry the build the individual project again." />
+
+    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true' and '$(BuildingInsideVisualStudio)'=='true'"
+      Text="The build can not be completed as the build tools package was just restored. Please reload the solution to allow targets from the build tools package to be imported." />
```

Comment:
@weshaggard I am telling them to reload the solution, you don't technically need to restart VS.
@nguerrera The projects load before the restore happens, you can see this if you try and surface something from one of the dependent imports or remove the condition. You wouldn't have seen this error before because it would get hit by one of the early (silent) design time builds. If you want to see the old error without this change you can turn on design time logging by setting TRACEDESIGNTIME=true in the environment. The logs will get dumped to the temp directory.